### PR TITLE
Improve table semantics using thead and tbody instead of a standard row as header

### LINKED ISSUE #670: [unlocalized strings] Sanitize English words before displaying, fix first row style
Realized on my demo server where the packages weren't installed yet, HTML code was displayed as part of the English string (and iframe was displayed).

We shouldn't also displaying the first row as an header (I had the same issue with the consistency table, so there's a rule do display the right color).

### PR DESCRIPTION
**Pull request by flodolo**
Strip tags in unlocalized view, that also improves results (e.g. for words like "tag", "file", etc.).
Also fixes #670

### CODE REVIEW

File: app/views/gaia.php [hunk 1]
```diff
@@ -217,16 +218,21 @@
...
-       . '</tr>'
-       . '<tr>'
-       . '<th>Key</th>'
-       . '<th>' . $repo_one . '</th>'
-       . '<th>' . $repo_two . '</th>'
-       . '</tr>';
 
+$section_title = "Strings that have changed significantly in English between {$repo_one} and {$repo_two} but for which the entity name didn’t change";
+$table = "
+    <p class='section_title'>{$section_title}</p>
```
Comment (pascalchevrel) [inline app/views/gaia.php]: If it's a title maybe it should use a <hx> tag ?

Comment (flodolo) [inline app/views/gaia.php]: Not completely sure: it's not an actual title, more a subtitle, and before it was even more confusing (part of the table).

schermata 2016-03-11 alle 13 54 01

Comment (flodolo) [inline app/views/gaia.php]: (it might make sense to change the class name, e.g. section_description or similar)

Comment (pascalchevrel) [inline app/views/gaia.php]: or many a <th> that would span all columns if it's part of the table, or a <caption> tag?

Comment (flodolo) [inline app/views/gaia.php]: Let me check how `<caption>` is displayed.

Comment (flodolo) [inline app/views/gaia.php]: Tried but it would require quite a few changes, especially for smaller viewports (it breaks quite badly).

**Comment by flodolo** [comment]
Updated variable/CSS class name in a separate commit. Not sure if we should drop the bold too.

**Comment by pascalchevrel** [comment]
oops, missed that it wasn't squashed yet :(

**Pull request merged**